src/constants.py

```diff
@@ -45,36 +44,6 @@
     f"/apps?appUrl=https://cloudflare-ipfs.com/ipfs/{CSV_APP_HASH}/"
 )
 SAFE_URL = f"{SAFE_URL}/{SHORT_NAME}:{SAFE_ADDRESS}/transactions/queue"
-# Things requiring dummy Web3 instance
-ERC20_ABI = json.loads(
```

Comment:
This hardcoded ABI constant has been moved into `abi/erc20.json` along with a more generic abi loader.